runtime/protocol/http-client-engines/http-client-engine-crt/common/src/aws/smithy/kotlin/runtime/http/engine/crt/CrtHttpEngine.kt

```diff
@@ -131,3 +144,32 @@ public class CrtHttpEngine(public val config: CrtHttpEngineConfig) : HttpClientE
         }
     }
 }
+
+internal suspend fun HttpStream.sendChunkedBody(body: HttpBody) {
```

Comment:
nit: This extension doesn't feel like it belongs in this file, maybe move to `RequestUtil.kt` or something.